frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+node_modules/
+.pnp/
+.pnp.js
+
+# testing
+/coverage
```

Comment:
do you use this folder?

Reply:
nope it's fom the template honestly I ignore what they are just Im not used to remove them in case... xd

Reply:
ok, it's normal. let's remove what is useless here